src/main/java/no/nav/fo/veilarbdialog/service/DialogDataService.java

```diff
@@ -189,8 +172,16 @@ private void oppdaterDialogTilHistorisk(DialogData dialogData) {
     }
 
     private DialogData sjekkLeseTilgangTilDialog(DialogData dialogData) {
-        pepClient.sjekkLesetilgangTilAktorId(dialogData.getAktorId());
+
+        if (!auth.harTilgangTilPerson(dialogData.getAktorId())) {
+            throw new IngenTilgang(String.format(
```

Comment:
Skal denne egentlig kaste exception? Blir usikker hvor denne kalles og hvordan det i forhold til det blir mtp kvp for eksempel nå men det er ikke nødvendigvis sånn at det skal komme en feilmelding når man ikke har tilgang til data man spør om fra frontend.

Reply:
Følger av gamle `PepClient.java:77` som gjør det samme dersom du ikke har tilgang. Om det er bra at det er sånn er en annen sak.